src/frontend/js/widgets/SyllabusCourseRunsList/index.spec.tsx

```diff
@@ -161,8 +169,12 @@ describe('<SyllabusCourseRunsList/>', () => {
     const course = CourseLightFactory().one();
     const courseRuns = [
       CourseRunFactoryFromPriority(Priority.FUTURE_NOT_YET_OPEN)().one(),
-      CourseRunFactoryFromPriority(Priority.FUTURE_CLOSED)().one(),
-      CourseRunFactoryFromPriority(Priority.ONGOING_CLOSED)().one(),
+      CourseRunFactoryFromPriority(Priority.FUTURE_CLOSED)({
+        resource_link: 'https://openedx.endpoint/course-v1:edX+DemoX+Session1/info/',
```

Comment:
why did you had to add this ?

Reply:
Because, now ongoing course runs (`FUTURE_CLOSED` + `ONGOING_CLOSED`) are rendering `CourseRunItemWithEnrollment` that checks enrollment. Under the hood they are using `findLMSBackend` method and we do not set resource_link properly, this method raise an error.